tests/integration/create_get_delete/integration_test.go

```diff
@@ -116,43 +142,49 @@ var _ = Describe("Create (Integration)", func() {
 		})*/
 
 		Context("and we create a deployment using kubectl", func() {
-			var (
-				kube *harness.Harness
-			)
+			var test *harness.Test
+
+			h := harness.New(harness.Options{
+				Kubeconfig: kubeconfigPath,
+				Logger:     &logger.TestLogger{},
+			})
 
 			BeforeEach(func() {
-				opts := harness.Options{
-					Kubeconfig: pathKubeCtlConf,
-					Logger:     &logger.TestLogger{},
-				}
-				kube = harness.New(opts)
+				err := h.Setup()
+				Expect(err).ShouldNot(HaveOccurred())
+				test = newKubeTest(h)
+				test.CreateNamespace(test.Namespace)
 			})
 
-			It("should deploy the service to the cluster", func() {
-				test := kube.NewTest(GinkgoT()).Setup()
-				defer test.Close()
+			AfterEach(func() {
+				test.Close()
+			})
 
-				d := test.CreateDeploymentFromFile("default", "nginx-deployment.yaml")
+			It("should deploy the service to the cluster", func() {
+				d := test.CreateDeploymentFromFile(test.Namespace, "podinfo.yaml")
 				test.WaitForDeploymentReady(d, 1*time.Minute)
 
-				// For each pod of the Deployment, check we receive a sensible response to a
-				// GET request on /.
-				for _, pod := range test.ListPodsFromDeployment(d).Items {
-					_, err := test.PodProxyGet(&pod, "80", "/").DoRaw()
+				pods := test.ListPodsFromDeployment(d)
+				Expect(len(pods.Items)).To(Equal(2))
+				// // For each pod of the Deployment, check we receive a sensible response to a
+				// // GET request on /.
+				// for _, pod := range pods.Items {
+				// 	_, err := test.PodProxyGet(&pod, "http", "/").DoRaw()
 
-					Expect(err).ShouldNot(HaveOccurred())
-					//TODO: compare response???
-				}
+				// 	Expect(err).ShouldNot(HaveOccurred())
```

Comment:
Do you think in the future its worth doing a test on a GET to the service? I think it would be good.

Reply:
We have it now, and actually validate the response too! 👍 😸